lib/arclight/custom_component.rb

```diff
@@ -46,9 +47,10 @@ def to_solr(solr_doc = {})
     def component_field_definitions
       [
         { name: 'level', value: formatted_level, index_as: :facetable },
-        { name: 'access_subjects', value: access_subjects, index_as: :facetable },
+        { name: 'access_subjects', value: access_subjects, index_as: :symbol },
         { name: 'containers', value: containers, index_as: :symbol },
-        { name: 'has_online_content', value: online_content?, index_as: :symbol }
+        { name: 'has_online_content', value: online_content?, index_as: :symbol },
+        { name: 'places', value: places, index_as: :facetable }
```

Comment:
I think this is redundant due to the terminology indexing it already. When you add a term to the terminology, it's automagically indexed. So since line 22 has `facetable` as one of the `index_as` parameters, I believe this is redundant.